litecommands-fabric/src/main/java/dev/rollczi/litecommands/fabric/FabricCommand.java

```diff
@@ -0,0 +1,148 @@
+package dev.rollczi.litecommands.fabric;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.builder.ArgumentBuilder;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.builder.RequiredArgumentBuilder;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.suggestion.SuggestionProvider;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import dev.rollczi.litecommands.argument.Argument;
+import dev.rollczi.litecommands.argument.parser.input.ParseableInput;
+import dev.rollczi.litecommands.argument.suggester.input.SuggestionInput;
+import dev.rollczi.litecommands.command.CommandRoute;
+import dev.rollczi.litecommands.command.executor.CommandExecutor;
+import dev.rollczi.litecommands.invocation.Invocation;
+import dev.rollczi.litecommands.join.JoinArgument;
+import dev.rollczi.litecommands.platform.PlatformInvocationListener;
+import dev.rollczi.litecommands.platform.PlatformSuggestionListener;
+import dev.rollczi.litecommands.quoted.QuotedStringArgumentResolver;
+import dev.rollczi.litecommands.suggestion.SuggestionResult;
+import net.minecraft.server.command.ServerCommandSource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.regex.Pattern;
+
+/**
+ * 2024/2/13<br>
+ * LiteCommands<br>
+ *
+ * @author huanmeng_qwq
+ */
+public class FabricCommand {
+    private static final Pattern PATTERN_ON_SPACE = Pattern.compile(" ", Pattern.LITERAL);
+
+    static List<FabricCommand> INSTANCES = new ArrayList<>();
```

Comment:
Move this to the platform implementation. Make this private and not static also `instances` -> `fabricCommands`